adoc/chapters/programming_interface.adoc

```diff
@@ -20154,8 +20154,8 @@ group.
 
 _Returns:_ the value of [code]#x# from the work-item whose group local id
 ([code]#id#) is [code]#delta# smaller than that of the calling work-item.
```

Comment:
Just looking at the sentence, is this concept of `delta larger` or `delta smaller` a well-known mathematical concept or could we make this clearer by using an equality instead?

Reply:
Resolve this consistently in another PR